credsweeper/app.py

```diff
@@ -76,10 +73,15 @@ def __init__(self,
         self.json_filename: Optional[str] = json_filename
         self.ml_batch_size = ml_batch_size
         self.ml_threshold = ml_threshold
-        if ml_validation:
+        if self._use_ml_validation():
             from credsweeper.ml_model import MlValidator
             self.ml_validator = MlValidator(threshold=self.ml_threshold)
 
+    def _use_ml_validation(self) -> bool:
+        if self.ml_threshold != 0:
```

Comment:
```suggestion
        if 0 < self.ml_threshold:
```
btw. negative values might be passed